app/src/main/java/com/example/sabonit/Account.java

```diff
@@ -28,7 +28,7 @@
     // User's phone number, may can be extracted from the google acount
     private String phoneNumber;
     // User's cart, empty when initialized
-    private Cart cart;
```

Comment:
instead of remove cart, add setter, i'll do it after your merging

Reply:
OK thanks